classes/music_album.rb

```diff
@@ -0,0 +1,14 @@
+require 'item'
+
+class MusicAlbum < Item
+  def initialize(_publish_date, archived, on_spotify)
+    super(id, published_date, archived)
+    @on_spotify = on_spotify
+  end
```

Comment:
I think it would be better if you use keyword arguments for `on_spotify`,`archived` property and type boolean.
`def initialize(_publish_date, archived: false, on_spotify: false)
  super(id, published_date, archived: archive)
    @on_spotify = on_spotify
end
`